modules/nupac.nu

```diff
@@ -211,16 +239,15 @@ export def "nupac install" [
     let add-to-config = (get-flag-value $add-to-config "NUPAC_ADD_TO_SCRIPTS_LIST")
```

Comment:
I wonder if removing the unnecessary state and handlig it just in L252 if would be better.

Reply:
anyway, gonna remove it and we can revert commit if needed